src/shared/components/ncTable/partials/rowTypePartials/TextRichForm.vue

```diff
@@ -11,13 +11,13 @@
 					<Fullscreen :size="20" />
 				</template>
 			</NcButton>
-			<NcModal v-if="showBigEditorModal" :close-button-contained="true" size="full" :title="column.title" @close="showBigEditorModal = false">
+			<NcModal v-if="showBigEditorModal" :close-button-contained="false" size="full" @close="showBigEditorModal = false">
 				<div class="row">
 					<div class="fix-col-4">
 						<NcEditor :text.sync="localValue" :show-border="false" />
 					</div>
 				</div>
-				<div class="closeModalButton">
+				<div class="closeModalButton row space-R">
```

Comment:
Haven't tested yet but how does the button align when there is long text so that the editor scrolls?